test/kwokctl/kwokctl_exec_test.sh

```diff
@@ -88,10 +90,28 @@ function main() {
     echo "------------------------------"
     echo "Testing exec on ${KWOK_RUNTIME} for ${release}"
     name="exec-cluster-${KWOK_RUNTIME}-${release//./-}"
-    create_cluster "${name}" "${release}" --config "${DIR}/exec.yaml"
+    if [[ "${KWOK_RUNTIME}" != "binary" ]]; then
+      yaml="${DIR}/exec-security-context.yaml"
+    else
+      yaml="${DIR}/exec.yaml"
+    fi
+    create_cluster "${name}" "${release}" --config - <<EOF
+apiVersion: config.kwok.x-k8s.io/v1alpha1
+kind: KwokConfiguration
+options:
+  enableCRDs:
+  - ClusterExec
+  - Exec
+EOF
+    create_user "${KWOK_RUNTIME}" "${release}" "kwok-controller" 1001 "test" 1002 "test" "/home/test" "/bin/sh"
```

Comment:
```suggestion
if [[ "${KWOK_RUNTIME}" != "binary" && "${KWOK_RUNTIME}" != "kind" && "${KWOK_RUNTIME}" != "kind-podman" ]]; then
    create_user "${KWOK_RUNTIME}" "${release}" "kwok-controller" 1001 "test" 1002 "test" "/home/test" "/bin/sh"
fi
```

kind and kind-podman should have failed the test, but why did it pass?